index.html

```diff
@@ -339,6 +339,22 @@ <h3>Success Criteria</h3>
             </li>
           </ul>
         </div>
+        <div>
+          <h3>Interoperability</h3>
```

Comment:
Having a section on interop is really strange for a W3C Charter... I don't know of any other charter that has one. For an extreme example, here's the web platform's charter -- look at the list of deliverables for the group, it's massive... note no section on interop: https://www.w3.org/2017/08/webplatform-charter.html (but the statements about interop are exactly the right sort of thing to say in a charter). W3C has a definition of interop, we don't need to elaborate so deeply on it.